src/Hw.java

```diff
@@ -0,0 +1,32 @@
+import java.util.*;
+
+public class Hw {
+    public static void main(String[] args) {
+        int[] array1 = {5, 11, 7, 20};
+
+        for (int i = 0; i < array1.length; i++) {
+            System.out.println(Arrays.stream(array1).min());
+        }
+        List<String> list = new ArrayList<>();
+
+        list.add("Енеїда");
```

Comment:
It's needed to create a method that will be do this transformation

Reply:
public static HashMap<Integer, String> createBookMap(ArrayList<String> bookList) {
        HashMap<Integer, String> bookMap = new HashMap<>();

        for (int i = 0; i < bookList.size(); i++) {
            bookMap.put(i, bookList.get(i));
        }
        return bookMap;
    }